packages/monitor/src/monitorHandler.ts

```diff
@@ -0,0 +1,172 @@
+import dotenv from 'dotenv'
+import chalk from 'chalk'
+import redis from './redisHandler'
+import { ethers } from 'ethers'
+import TelegramBot from 'node-telegram-bot-api'
+import { markets, executorAddresses, rpcURL, OVTokenAddress, erc20ABI} from './constants'
+
+dotenv.config()
+
+const log = console.log
+
+interface MarketData {
+  totalLiquidatedPositions: number
+  liquidatedPositions: number
+  liquidatablePositionsFound: number
+}
+
+interface ExecutorData {
+  totalLiquidatedPositions: number
+  liquidatedPositions: number
+  ethBalance: string
+  ovBalance: string
+  ovRewardsClaimed: string
+}
+
+interface LiquidatorStats {
+  totalLiquidatedPositions: number
+  liquidatedPositions: number
+  liquidatablePositionsFound: number
+  dataByMarket: { [key: string]: MarketData }
+  dataByExecutor: { [key: string]: ExecutorData }
+  prevTimestamp: string
+}
+
+async function sendTelegramMessage(message: string) {
+  if (!process.env.TELEGRAM_BOT_TOKEN || !process.env.TELEGRAM_CHAT_ID) {
+    log(chalk.bold.red('TELEGRAM_BOT_TOKEN and TELEGRAM_CHAT_ID must be provided'))
+    return
+  }
+
+  const bot = new TelegramBot(process.env.TELEGRAM_BOT_TOKEN, { polling: false })
```

Comment:
`TelegramBot` will be initialized every time we send a message. Will it make sense to initialize `bot` variable in global space and use `||=` to initialize/assign?

Reply:
done!